scripts/dump_local_db.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+#################
+# Setup Variables
+#################
+
+if [ -z "$DATABASE_URL" ]; then
+    echo "DATABASE_URL must be a valid connection URL. Are you in the docker environment?"
+    exit 1
+fi
+
+# Parse DATABASE_URL for database connection parts
```

Comment:
Note: copied from `scripts/create_sample_db.sh`. I'm not sure what the function reuse story is for bash.